spacer/tasks.py

```diff
@@ -28,8 +27,14 @@ def extract_features(msg: ExtractFeaturesMsg) -> ExtractFeaturesReturnMsg:
 
     check_extract_inputs(img, msg.rowcols, msg.image_loc.key)
 
+    check_rowcols(msg.rowcols, img)
```

Comment:
`check_rowcols()` was replaced with `check_extract_inputs()` in PR #70. So don't need this line.